src/main.cpp

```diff
@@ -1,6 +1,23 @@
+#include <SFML/Graphics.hpp>
+#include <SFML/Window.hpp>
 #include <iostream>
 
 int main() {
-	std::cout << "Hello world!\n" << std::endl;
-	return 0;
+	sf::RenderWindow window(sf::VideoMode(800, 600), "SFML window");
+	window.setVerticalSyncEnabled(true);
+	window.setFramerateLimit(60);
+
+	while (window.isOpen()) {
+		sf::Event event;
+		while (window.pollEvent(event)) {
+			switch (event.type) {
+			case sf::Event::Closed: window.close(); break;
+			case sf::Event::KeyPressed: std::cout << "ping\n"; break;
+			default: break;
+			}
+		}
+		window.clear();
+		window.display();
+	}
+	return EXIT_SUCCESS;
```

Comment:
Just FYI (no need to change): in C++ we usually just write `0` (or omit `return 0` entirely).

Reply:
Yeah, I just wanted to be a bit explicit. This will most likely go away anyway.